.coderabbit.yaml

```
@@ -22,11 +22,16 @@ reviews:
     - path: "**"
       instructions: |
         Pull requests that have "TECH_DEBT" in the title should only contain changes related to typos, unused code, linter/IDE suggestions, swagger specification updates,
-        documentation (such as markdown files) and logging improvements. These TECH_DEBT PRs must not affect core functionality. All PRs that are not considered
-        technical debt must include information on what testing was performed in the description of the PR. If it does not, ask the author to provide details on
-        what testing was performed.
+        and logging improvements. These TECH_DEBT PRs must not affect core functionality. All PRs that are not considered technical debt must include information on what 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)